### DISCUSSION TITLE
What is the proper way to report an issue with GitHub actions?

### DISCUSSION THREAD

**Discussion by druzm** [community/community#56412] [Actions]
### Select Topic Area

Question

### Body

Hello everyone,

I'd like to know what the proper way of reporting an issue with GitHub Actions is. [githubstatus.com](githubstatus.com) shows no issues, and no problems have been reported through the associated Twitter account. However, when I push code to my repo which triggers the CI, the tests are only run for ubuntu-latest, whereas the windows-latest and macOS-latest tests end up stuck in the "Test with pytest" step of my script, and the tests are never run.

The issue began at least last week, and has been present most of the time. Three days ago, and then yesterday, it seemed to have disappeared completely but it is back now.

**Comment by druzm** [comment]
Never mind, this seems to be the right link: https://support.github.com/contact/bug-report

**Comment by frankvelezf** [comment]
Japan or Asia rules
